### PR TITLE
Fixes to account representation in frontend

### PR DESCRIPTION
**Pull request by eyevz**
There were several problems in type interfaces and buffer layouts, affecting Reserves. Hopefully this PR will address some of  the confusing behaviour of our devnet deployment.

### CODE REVIEW

**Comment by eyevz** [comment]
Integration tests are a work-in-progress. I've been modifying them to dig into interest charges and haven't resolved that yet.

File: app/src/models/JetTypes.ts [hunk 1]
```diff
@@ -157,15 +160,24 @@ export interface ReserveConfigStruct {
...
+  /** Maximum number of tokens to sell in a single DEX trade during liquidation */
+  liquidationDexTradeMax: number,
+  /** Unused space */
+  _reserved1: number[],
 };
+
 export type ReserveStateStruct = CacheStruct & {
   accruedUntil: BN,
-  oustandingDebt: BN,
+  outstandingDebt: BN,
```
Comment (Tristyn) [inline app/src/models/JetTypes.ts:175]: I bet it's 'outstandingDebt' in the buffer layout. This should fix the outstandingDebt being zero in frontend

File: app/src/models/JetTypes.ts [hunk 2]
```diff
@@ -118,15 +117,19 @@ export interface ReserveAccount {
...
   depositNoteMint: PublicKey,
   loanNoteMint: PublicKey,
   vault: PublicKey,
   feeNoteVault: PublicKey,
+  dexSwapTokens: PublicKey,
+  dexOpenOrders: PublicKey,
+  dexMarket: PublicKey,
+  _reserved0: number[],
   config: ReserveConfigStruct,
-  accruedUntil: BN,
```
Comment (Tristyn) [inline app/src/models/JetTypes.ts:128]: Caught accruedUntil with Eric earlier. Nice work

File: app/src/models/JetTypes.ts (hunk 1 above)
Comment (GHesericsu) [reply app/src/models/JetTypes.ts:175]: ah I thought I was the first to fix this lol. beat me to it.

Comment (GHesericsu) [reply app/src/models/JetTypes.ts:175]: same typo on subscribe.ts. I will just push that change later.

**Comment by Tristyn** [comment]
Good to merge though tests are failing

**Pull request merged**